src/openrct2-ui/windows/GuestList.cpp

```diff
@@ -928,7 +928,9 @@ static FilterArguments get_arguments_from_peep(const Peep* peep)
             {
                 std::memset(gCommonFormatArgs, 0, sizeof(gCommonFormatArgs));
                 peep_thought_set_format_args(thought);
```

Comment:
This function should take the formatter ft we shouldn't be playing around setting gCommonFormatArgs to zero or anything else.